conan/build.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from conan.packager import ConanMultiPackager
+import os, re
+
+
+def get_value_from_recipe(search_string):
+    with open("conanfile.py", "r") as conanfile:
+        contents = conanfile.read()
+        result = re.search(search_string, contents)
+    return result
+
+
+def get_name_from_recipe():
+    return get_value_from_recipe(r'''name\s*=\s*["'](\S*)["']''').groups()[0]
+
+
+def get_version_from_recipe():
+    return get_value_from_recipe(r'''version\s*=\s*["'](\S*)["']''').groups()[0]
+
+
+def running_on_appveyor():
+    return os.getenv("APPVEYOR", False)
+
+
+def get_env_vars():
+    username = os.getenv("CONAN_USERNAME", "flatbuffers")
+    channel = os.getenv("CONAN_CHANNEL", "testing")
+    name = get_name_from_recipe()
+    version = get_version_from_recipe()
+    return name, version, username, channel
+
+
+def get_stable_branch_pattern():
+    return "master" if running_on_appveyor() else r"v\d+\.\d+\.\d+"
+
+
+def set_appveyor_environment():
+    compiler_version = os.getenv("CMAKE_VS_VERSION").split(" ")[0].replace('"', '')
+    os.environ["CONAN_VISUAL_VERSIONS"] = compiler_version
+
+    ci_platform = os.getenv("Platform").replace('"', '')
+    ci_platform = "x86" if ci_platform == "x86" else "x86_64"
+    os.environ["CONAN_ARCHS"] = ci_platform
+
+    os.environ["CONAN_BUILD_TYPES"] = os.getenv("Configuration").replace('"', '')
+
+
+def add_visual_build(builder, version, arch, build_type, runtime, shared):
+    settings = {'compiler.version': version, 'arch': arch, 'build_type': build_type,
+                'compiler.runtime': '%s%s' % (runtime, ('d' if build_type == "Debug" else '')),
```

Comment:
Runtime might be simplified if doing only MD/MDd, as now conan can dynamically adjust it to release/debug according to ``build_type``

Reply:
The point here is that conan-package-tools doesn't solve x86_64 for Visual Studio 10. Would be nice to solve all this stuff directly from package tools.

Reply:
My fault! I just revised package tools and I found vs10_x86_64_enabled. I'll remove this block.